examples/funcalign/searchlight_srm_example.py

```diff
@@ -0,0 +1,152 @@
+#  Copyright 2016 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# Author: Hejia Zhang (Princeton University ELE Department)
+
+"""Distributed Searchlight Example (if run 16 ranks)
+example usage: mpirun -n 16 python3 searchlight_srm_example.py
+"""
+
+import numpy as np
+from mpi4py import MPI
+import sys
+import scipy.io as sio
+from scipy.stats import stats
+from brainiak.searchlight.searchlight import Searchlight
+from brainiak.funcalign.srm import SRM
+import warnings
+
+# parameters
+sl_rad = 1 #searchlight length (of each edge) will be 1+2*sl_rad
+max_blk_edge = 1 #won't change computational results, has effect on MPI processing
```

Comment:
You should leave `max_blk_edge` out, unless you want to provide some insight into its impact on performance. Searchlight will use the default value if you do not provide it.